replica-common/src/main/java/com/graphhopper/reader/osm/CustomOsmReader.java

```diff
@@ -358,6 +364,7 @@ else if (config.getElevationSmoothing().equals("moving_average"))
 
         checkDistance(edge);
         restrictedWaysToEdgesMap.putIfReserved(way.getId(), edge.getEdge());
+        ghEdgeIdToSegmentIndex.put(edge.getEdge(), segmentIndex);
```

Comment:
This is (finally) where we actually store the raw mapping of <gh edge ID> -> <segment index>

Reply:
I clearly needed more coffee before writing these comments 😅 but, yes!